common/rdm/AckTimerResponder.cpp

```diff
@@ -208,13 +208,29 @@ RDMResponse *AckTimerResponder::ResponseFromQueuedMessage(
     const class QueuedResponse *queued_response) {
   switch (queued_response->CommandClass()) {
     case RDMCommand::GET_COMMAND_RESPONSE:
+      return new RDMGetResponse(
+        // coverity[SWAPPED_ARGUMENTS]
+        request->DestinationUID(),
+        request->SourceUID(),
+        request->TransactionNumber(),
+        RDM_ACK,
+        QueuedMessageCount(),
+        ROOT_RDM_DEVICE,
+        queued_response->Pid(),
+        queued_response->ParamData(),
+        queued_response->ParamDataSize());
     case RDMCommand::SET_COMMAND_RESPONSE:
-      return GetResponseFromData(request,
```

Comment:
Not just that, a GET_REQUEST can reply with a SET_RESPONSE. Queued messages are funky.